WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -196,6 +196,15 @@ final class PostListViewController: AbstractPostListViewController, UIViewContro
             return
         }
 
+        if let error = PostCoordinator.shared.syncError(for: post.original()),
```

Comment:
Please move it to `PostListEditorPresenter` and also add it to the `PagetListEditorPresenter` and test that it works in search.

Reply:
Fixed! https://github.com/wordpress-mobile/WordPress-iOS/pull/22992/commits/e3a7a868f66185658fb4e5bd14ea74dfc949bc6f

This is now working as expected for post list, page list, and search